src/components/DropdownSelect.js

```diff
@@ -9,69 +9,92 @@ import { Wrapper, WrapperRow, WrapperHorizontal } from './Wrapper';
 import { Icon } from './Icon';
 import { arrowDown, arrowUp } from '../icons';
 import { Label } from './Label';
+import { OrientationContext } from '../OrientationProvider';
 
-export const DropdownSelect = memo(
-  ({ data, setData, label, showSearch, searchInputStyle, renderSearch, searchPlaceholder }) => {
-    if (!data || !data.length) return null;
+export const DropdownSelect = ({
+  data,
+  setData,
+  label,
+  showSearch,
+  searchInputStyle,
+  renderSearch,
+  searchPlaceholder
+}) => {
+  const { orientation } = useContext(OrientationContext);
 
-    const [arrow, setArrow] = useState('down');
-    const selectedData = data.find((entry) => entry.selected);
-    const selectedIndex = data.findIndex((entry) => entry.selected);
-    const preselect = (index) => this[`dropdown${label}`].select(index);
+  const marginHorizontal =
+    orientation === 'portrait' ? normalize(14) : device.height * 0.15 + normalize(14);
 
-    useEffect(() => {
-      preselect(selectedIndex);
-    }, [selectedData]);
+  const adjustFrame = useCallback(
```

Comment:
this adjusts the dropdown modal on orientation change now?

Reply:
the marginHorizontal inside (which is set 2 lines above) is the deciding factor, yes

Reply:
see `adjustFrame` here https://github.com/siemiatj/react-native-modal-dropdown#props